src/puppet-wechat4u.ts

```diff
@@ -225,6 +250,48 @@ export class PuppetWechat4u extends PUPPET.Puppet {
     wechat4u[func] = funcNew
   }
 
+  /**
+   * @private
+   * For issue #668
```

Comment:
Could you please add the issue link under this line?

Reply:
I have created another PR to complete this question